tests/Integration/Pagination/PaginateDirectiveDBTest.php

```diff
@@ -424,6 +424,96 @@ public function testQueriesPaginationWithNoData(): void
         ])->assertJsonCount(0, 'data.users.data');
     }
 
+    public function testQueriesPaginationWithoutPaginatorInfo(): void
+    {
+        $user = factory(User::class)->create();
+
+        $this->schema /** @lang GraphQL */
+            = '
+        type User {
+            id: ID!
+        }
+
+        type Query {
+            users: [User!]! @paginate
+        }
+        ';
+
+        DB::enableQueryLog();
```

Comment:
We typically use another method to count database queries, something with registering a query listener. Let's stay consistent - perhaps abstract?

Reply:
I copied this from another test in the exact same file...

But I'll browse around then.

Reply:
Got it. If you could unify the approaches that would be great.

Reply:
I have checked around but was unable to find anywhere we are query counting.

Maybe I'm blind but the `Event::fake` route also wasn't working for me so took a trait that add some nice syntax sugar for future query counting needs :)